steps/google-drive-uploader/0.0.1/step.yml

```diff
@@ -0,0 +1,57 @@
+title: google-drive-uploader
+summary: |
+  Uploads apks generated to google drive
+description: |
+  Using the google drive api we can upload apks to a specified google drive folder
+website: https://github.com/FriendlyUser/bitrise-step-google-drive-uploader
+source_code_url: https://github.com/FriendlyUser/bitrise-step-google-drive-uploader
+support_url: https://github.com/FriendlyUser/bitrise-step-google-drive-uploader/issues
+published_at: 2022-11-25T20:30:21.169363-08:00
+source:
+  git: https://github.com/FriendlyUser/bitrise-step-google-drive-uploader.git
+  commit: 4e64883e28d432fc1c1829f3c8ae5e7fe88e4381
+type_tags:
+- utility
+toolkit:
+  go:
+    package_name: github.com/FriendlyUser/bitrise-step-google-drive-uploader
+inputs:
```

Comment:
Suggestion: an additional Step input could collect the file path of the APK to be uploaded.